frontend/src/app/application-forms/fields/activity-description.component.html

```diff
@@ -4,15 +4,15 @@
 
     <label id="number-of-trips-label" class="usa-input">The anticipated number of trips. <span class="required-fields-asterisk">*</span></label>
     <span id="participants-hint-text" class="help-text usa-form-hint">Please provide the number of trips anticipated (Note: The use of a dash is permitted to supply a possible range).</span>
-    <input class ="number-input-sizing" id="number-of-trips"  type="tel" pattern="[0-9\-]+" maxlength="7"formControlName="numberOfTrips" aria-required="true"
+    <input class ="number-input-sizing" id="number-of-trips"  type="tel" pattern="[0-9\-]+" maxlength="7" formControlName="numberOfTrips" aria-required="true"
```

Comment:
input type should still be "text". we're not looking for a tel input. but the regex pattern is correct and will handle correct inputs. tested locally to work.